examples/design-system/README.md

```diff
@@ -1,53 +1,189 @@
-# Turborepo Design System starter
+# Turborepo Design System Starter
 
-This is an official React design system starter powered by Turborepo.
+This guide explains how to use a React design system starter powered by:
 
-## What's inside?
+- 🏎 [Turborepo](https://turborepo.org) — High-performance build system for Monorepos
+- 🚀 [React](https://reactjs.org/) — JavaScript library for user interfaces
+- 🛠 [Tsup](https://github.com/egoist/tsup) — TypeScript bundler powered by esbuild
+- 📖 [Storybook](https://storybook.js.org/) — UI component environment powered by Vite
 
-This Turborepo includes the following packages and apps:
-
-### Apps and Packages
-
-- `docs`: A placeholder documentation site powered by [Next.js](https://nextjs.org)
-- `@acme/core`: core React components
-- `@acme/utils`: shared React utilities
-- `@acme/tsconfig`: shared `tsconfig.json`s used throughout the monorepo
-- `eslint-config-acme`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
-
-Each package and app is 100% [TypeScript](https://www.typescriptlang.org/).
-
-### Utilities
-
-This turborepo has some additional tools already setup for you:
+As well as a few others tools preconfigured:
 
 - [TypeScript](https://www.typescriptlang.org/) for static type checking
 - [ESLint](https://eslint.org/) for code linting
 - [Prettier](https://prettier.io) for code formatting
+- [Changesets](https://github.com/changesets/changesets) for managing versioning and changelogs
+- [GitHub Actions](https://github.com/changesets/action) for fully automated package publishing
 
-## Using this example
+## Getting Started
 
-We do not have a starter yet in `create-turbo` for this quite yet. If you want to use this in the interim, you run the following command:
+Clone the design system example locally or [from GitHub](https://github.com/vercel/turborepo/tree/main/examples/with-changesets):
 
-```sh
-npx degit vercel/turborepo/examples/design-system design-system
+```bash
+npx degit vercel/turborepo/examples/with-changesets design-system
 cd design-system
 yarn install
-git init . && git add . && git commit -m "Init"
 ```
```

Comment:
IMO we don't need this line, open to other ideas though.

Reply:
We actually should keep it. Turborepo runs faster with git initialized; we leverage `git ls-tree` to get git's precomputed hashes for files. Without git, `turbo` has to manually hash the contents of each file which is usually much slower.

Reply:
SGTM